### PR TITLE
ctrl-c: add recipe

### PR DESCRIPTION
**Pull request by toge**
### Summary
Changes to recipe:  **ctrl-c/***

#### Motivation
I want to use this library to process Ctrl-C signal.

#### Details
https://github.com/evgenykislov/ctrl-c

---
- [x] Read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md)
- [x] Checked that this PR is not a duplicate: [list of PRs by recipe](https://github.com/conan-io/conan-center-index/discussions/24240)
- [x] Tested locally with at least one configuration using a recent version of Conan

### CODE REVIEW

File: recipes/ctrl-c/all/conandata.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+sources:
+  "cci.20240805":
```
Comment (AbrilRBS) [inline recipes/ctrl-c/all/conandata.yml:2]:
```suggestion
  "0.0.0.cci.20240805":
```

Comment (AbrilRBS) [reply recipes/ctrl-c/all/conandata.yml:2]: (and similarly in the `config.yml` file

**Review by AbrilRBS — COMMENTED** [review]
Looks good, only a minor change needed :)

**Review by uilianries — APPROVED** [review]
LGTM. The upstream does not offer a cmakelists, but sources are simple enough.

**Pull request merged**